### ISSUE TITLE
Adding firewall rule fails with version v1.43.0

### ISSUE THREAD

**Issue by Dari4sho** [hetznercloud/cli#725]
### TL;DR

Starting from v1.43.0, adding a firewall rule fails with `hcloud: invalid input in field 'rules' (invalid_input)`

### Expected behavior

Firewall rule being added successfully

### Observed behavior

Adding firewall rule fails with  `hcloud: invalid input in field 'rules' (invalid_input)`

### Minimal working example

I am running hcloud on a scheduled pipeline always using the latest version.
Since the release of v1.43.0 yesterday, adding a firewall rule fails for me.

This is the command that I run:
`hcloud firewall add-rule 1234567 --direction=in --port=2205 --protocol=tcp --source-ips=12.34.56.78/32 --description="Temp access"`

It works fine with version v1.42.0.

### Log output

```plain text
`hcloud: invalid input in field 'rules' (invalid_input)`
```

### Additional information

Fails on version v1.43.0
Works on version v1.42.0

**Comment by apricote** [comment]
Fixed in [v1.43.1](https://github.com/hetznercloud/cli/releases/tag/v1.43.1).

Thanks for the report @Dari4sho!
